infrabox/test/api/job_api_test.py

```diff
@@ -0,0 +1,183 @@
+from os import getcwd, stat, remove
+
+from pyinfraboxutils.token import encode_job_token
+from pyinfraboxutils.storage import storage
+from temp_tools import TestClient
+from test_template import ApiTestTemplate
+
+
+class JobApiTest(ApiTestTemplate):
+
+    url_ns = 'api/job'
+
+    #def test_uuid_validation(self):
+    #   pass
+    #   #TODO
```

Comment:
remove TODO or add a message, otherwise it will pop up in some editors without meaningful information